arches/install/arches-templates/.gitignore

```
@@ -1,16 +1,15 @@
 *.pyc
 *.log
+node_modules
 {{ project_name }}/logs
 {{ project_name }}/export_deliverables
 {{ project_name }}/cantaloupe/*
 {{ project_name }}/staticfiles
 {{ project_name }}/media/packages
-{{ project_name }}/media/node_modules
 {{ project_name }}/media/build/
 {{ project_name }}/uploadedfiles/*
 {{ project_name }}/settings_local.py
-{{ project_name }}/webpack/webpack-stats.json
-{{ project_name }}/webpack/webpack-user-config.js
+webpack-stats.json
```

Unless I have this backwards and it's the others that need changing?
```suggestion
webpack/webpack-stats.json
```

---

Oh word, I didn't realize that the gitignore is recursive, so this is totally fine. 👍